ONSdigital/eq-survey-runner
# Eq 1945 social alpha

### What is the context of this PR?
Author a first cut survey schema for Labour Force Survey.  This PR adds a new schema for LFS to `data/en`, there are no code changes.

Only the first path of the LFM design has been implemented (Socio Demographic), the remainder (Labour Market) will be implemented in a subsequent iteration.  Issues and omissions have been documented on the related [Trello card](https://trello.com/c/eBUSUODf).

### How to review 
- Check schema is valid using the schema validator i.e:
   `pipenv run ./scripts/test_schemas.sh data/en/labour_force.json`
- Run the survey [with this url](https://preprod-surveys-launch.eq.ons.digital/quick-launch?url=https://raw.githubusercontent.com/ONSdigital/eq-survey-runner/eq-1945-social-alpha/data/en/labour_force.json).
